sync3/caches/global.go

```diff
@@ -285,6 +285,10 @@ func (c *GlobalCache) OnNewEvent(
 		if ed.StateKey != nil && *ed.StateKey == "" {
 			metadata.NameEvent = ed.Content.Get("name").Str
 		}
+	case "m.room.avatar":
+		if ed.StateKey != nil && *ed.StateKey == "" {
+			metadata.AvatarURL = ed.Content.Get("avatar_url").Str
```

Comment:
`metadata.AvatarEvent` is a) more consistent with `NameEvent` and b) clarifies that this isn't the calculated avatar URL for the room.

Reply:
Also this key name is wrong, it should be `url` not `avatar_url`.

Reply:
2de017e